CONTRIBUTING.md

```diff
@@ -23,7 +23,6 @@
 >
 > _Director, ADL Initiative_
 >
-> _OSD, Training Readiness & Strategy (TRS)_
```

Comment:
I don't know what this is, so I'm curious now that it's being dropped. What is it?